plugins/modules/template_workflow_manager.py

```diff
@@ -1368,14 +1385,14 @@ def get_template_info(self, template_params):
                     if max_value is not None:
                         _range[j].update({"maxValue": max_value})
                     else:
-                        self.msg = "max_value is mandatory for range under template_params"
+                        self.msg = "max_value is required for range under template_params"
                         self.status = "failed"
                         return self.check_return_status()
                     min_value = value.get("min_value")
                     if min_value is not None:
                         _range[j].update({"minValue": min_value})
                     else:
-                        self.msg = "min_value is mandatory for range under template_params"
+                        self.msg = "min_value is required for range under template_params"
```

Comment:
The parameter 'min_value' is required for range under 'template_params' but not provided..